conf/modules.config

```diff
@@ -42,32 +43,29 @@ process {
     //
 
     if("delly" in callers){
-        withName: '^.*:BAM_VARIANT_CALLING_DELLY:BEDTOOLS_SPLIT\$' {
-            ext.args = "-n ${params.delly_scatter_count}"
-        }
-
-        withName: DELLY_CALL {
+        withName: '^.*:BAM_VARIANT_CALLING_DELLY:DELLY_CALL\$' {
             ext.args = {"--svtype ${params.delly_sv_types}"}
+            ext.suffix = "vcf"
         }
```

Comment:
stringency parameters for delly
`        withName: DELLY_CALL {
            ext.args = {"--svtype ${params.delly_sv_types}"}
            ext.args = {"--svtype ${params.delly_sv_types}"} --map-qual ${params.delly_map_qual}"} --min-clique-size ${params.delly_min_clique_size}"}
        }
        }`